modules/tinymce/src/core/test/ts/browser/FormatterApplyTest.ts

```diff
@@ -2269,15 +2269,15 @@ describe('browser.tinymce.core.FormatterApplyTest', () => {
 
   it('Apply format to node outside fake table selection', () => {
     const editor = hook.editor();
-    editor.setContent('<p>test</p><table><tbody><tr><td data-mce-selected="1">cell 1</td><td>cell 2</td></tr><tr><td data-mce-selected="1">cell 3</td><td>cell 4</td></tr></tbody></table>');
```

Comment:
nodeChange-event makes this selection not work in this situation due to normalization.